src/_scss/components/_picker.scss

```diff
@@ -0,0 +1,48 @@
+.usa-dt-picker__list {
```

Comment:
Are these styles overriding the component library wherever we use the picker? Should we update the library instead?

Reply:
Yea kinda natural overrides IMO